test/DynamicDependency/data/Main.Sidecar/winmain.cpp

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+
+#include <IDynamicDependencyLifetimeManager.h>
+
+using namespace winrt;
+
+// Implement the LifetimeManager as a classic COM Out-of-Proc server, via C++/WinRT
+// See https://docs.microsoft.com/en-us/windows/uwp/cpp-and-winrt-apis/author-coclasses for more details
+
+static constexpr GUID MyLifetimeManager_guid // {32E7CF70-038C-429a-BD49-88850F1B4A11}
+{
+    0x32e7cf70, 0x038c, 0x429a, { 0xbd, 0x49, 0x88, 0x85, 0x0f, 0x1b, 0x4a, 0x11 }
+};
+
+struct MyLifetimeManager : winrt::implements<MyLifetimeManager, IDynamicDependencyLifetimeManager>
+{
+    HRESULT __stdcall Initialize()
+    {
+        return S_OK;
+    }
+
+    HRESULT __stdcall Shutdown()
+    {
+        return S_OK;
+    }
+
+    HRESULT __stdcall GetPackageFullName(/*[out, retval]*/ PWSTR* packageFullName)
+    {
+        *packageFullName = nullptr;
+
+        WCHAR fullName[PACKAGE_FULL_NAME_MAX_LENGTH + 1]{};
+        UINT32 fullNameLength = _countof(fullName);
```

Comment:
Guzak tells me this should be ARRAYSIZE as _countof is an unevenly-supported extension.

Reply:
Makes sense. I regularly see folks do _countof (or other similar non-ARRAYSIZE). I'll point folks toward ARRAYSIZE going forward.